app/views/groups/samples/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render Viral::PageHeaderComponent.new(title: t(:'.title'), subtitle: t(:'.subtitle', namespace_type: @group.class.model_name.human, namespace_name: @group.name)) do |component| %>
+  <%= component.with_icon(name: "beaker", classes: "h-14 w-14 text-primary-700") %>
+<% end %>
+
+<div class="p-4">
+  <div class="grid gap-4 xl:grid-cols-2 2xl:grid-cols-3">
```

Comment:
I think we probably want this more like the other tables in the IRIDA Next where they take up the full area.  I don't know what other information we want in this table but I assume it will be more than just name and project.  @ericenns or @deepsidhu85 thoughts on this?

Reply:
Yeah we would want this table to take up the full area like the rest of the tables. We will for sure want to display more information eventually such as QA status etc

Reply:
Please see 96c612e8a69723dac36a48be26c23509e5eef35b.